### PR TITLE
Small styling fixes

### PR DESCRIPTION
**Pull request by nellileinonen**
#### What are the main changes in this PR?
New styling for h3 and h4 headings and better alignment of content in some postboxes.

##### Why are we doing this? Any context or related work?
Issue: #389
Uniform content alignment throughout the plugin makes the UI feel more professional.

#### Manual testing steps?
Look headings for example in "Seravo Updates" postbox.
Look content alignment for example in "Backups", "Unnecessary Themes" and "Mails" postboxes.

#### Screenshots
Screenshot from 2020-07-13 15-03-05
Screenshot from 2020-07-13 15-02-17

### CODE REVIEW

File: style/seravo-postbox.css [hunk 1]
```diff
@@ -35,6 +35,14 @@
...
 }
 
+#dashboard-widgets h3 {
+  font-size: 130%;
+}
+
+#dashboard-widgets h4 {
+  font-size: 120%;
+}
+
```
Comment (elguitar) [inline style/seravo-postbox.css:45]: These are too broad, since there are h3 under dashboard-widgets on the WordPress front page too.
kuva

Comment (nellileinonen) [reply style/seravo-postbox.css:45]: You're right. I'll fix those to affect only seravo-postbox contents

File: style/seravo-postbox.css [hunk 2]
```diff
@@ -35,6 +35,14 @@
...
 }
 
+.seravo-postbox h3 {
+  font-size: 130%;
+}
+
+.seravo-postbox h4 {
+  font-size: 120%;
+}
+
```
Comment (elguitar) [inline style/seravo-postbox.css:45]: Sadly, now these are overriden by WordPress styles
kuva

Comment (nellileinonen) [reply style/seravo-postbox.css]: Ah, of course. Should have checked that more carefully, sorry for the extra work!

**Pull request merged**